### PR TITLE
pacman: use color by default

### PR DESCRIPTION
**Pull request by 1480c1**

### CODE REVIEW

**Comment by Biswa96** [comment]
1. Can space checking also disabled?
2. Can parallel download be enabled?

**Comment by jeremyd2019** [comment]
> 1. Can space checking also disabled?

https://github.com/msys2/setup-msys2/blob/master/main.js#L252 but for end-users it seems like a useful thing.

> 2. Can parallel download be enabled?

It already is.  https://github.com/msys2/MSYS2-packages/blob/master/pacman/pacman.conf#L36

**Comment by Biswa96** [comment]
> but for end-users it seems like a useful thing.

Everybody disable it in CI.

**Comment by 1480c1** [comment]
@lazka any opinion on the disabling checkspace? I personally found it a bit annoying due to how long it takes sometimes

**Comment by lazka** [comment]
> > but for end-users it seems like a useful thing.
> 
> Everybody disable it in CI.

because CI systems don't keep state, so a broken installation doesn't matter.

> @lazka any opinion on the disabling checkspace? I personally found it a bit annoying due to how long it takes sometimes

I remember it being widely different on some systems, it's pretty much instant here.

I'm not sure really.. I'll try testing what happens if there isn't enough disk space.

File: pacman/PKGBUILD [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 pkgname=pacman
 pkgver=6.0.0
-pkgrel=2
+pkgrel=3
```
Comment (lazka) [inline pacman/PKGBUILD:7]: update checksums

**Comment by 1480c1** [comment]
We can probably talk about disabling space check in another PR and keep this just to color

**Pull request merged**